OSDOCS-16940: DITA abstracts — OLM admin/user topics (#115359)
openshift/openshift-docs: modules/olm-deleting-operators-from-a-cluster-using-cli.adoc

[id="olm-deleting-operator-from-a-cluster-using-cli_{context}"]
= Deleting Operators from a cluster using the CLI

[role="_abstract"]

> To remove an installed Operator from a {product-title} namespace, you can delete its subscription and cluster service version (CSV) by using the CLI. You must remove the Operator completely before you reinstall it.

Reviewer: ```suggestion
To remove an installed Operator from a namespace, cluster administrators can delete its subscription and cluster service version (CSV) by using the CLI.
```
